Blockchain/readme.md

```diff
@@ -21,6 +21,12 @@
 - [Road to Web3](https://www.web3.university/tracks/road-to-web3)
 - [One of the best Web3 Roadmap](https://vitto.cc/web3-and-solidity-smart-contracts-development-roadmap/)
 
+### Additional resources for Blockchain Fundamentals
+- [Metaschool: Free cources](https://metaschool.so/courses)
+- [Lumos Academy: Learning Platform](https://academy.lumoslabs.co/courses)
+- [DeFi Infrastructure(Youtube)](https://www.youtube.com/playlist?list=PLE1Vu6ctbqa61FqJmPDnoPzDqrnyILHRm)
+- [Learn by reading Blogs](https://parthmandale.hashnode.dev/series/blockchain-defi) 
```

Comment:
```suggestion
```
We only want to include credible resources.

Reply:
Sir trust me those blog series are awesome, more blogs are going to be added to it. So it will be a beginner to advance series and anyone can learn from it in proper sequence. 
@siddhant-khisty 🙏 please let that be in additional resources.

Reply:
@siddhant-khisty Please merge my PR!

Reply:
will you please respond?

Reply:
@siddhant-khisty Im confused about what to do please suggest!